native/cocos/scene/Model.h

```diff
@@ -95,6 +95,7 @@ class Model : public RefCounted {
 
     virtual void updateTransform(uint32_t stamp);
     virtual void updateUBOs(uint32_t stamp);
+    void updateOctree();
```

Comment:
Why Model has a function to update octree?

Reply:
model update itself in octree